### ISSUE TITLE
Death Chest label lingers after chest is removed

### ISSUE THREAD

**Issue by sean-work** [TheRandomLabs/VanillaDeathChest#42]
![image](https://user-images.githubusercontent.com/68880081/94878100-5719ba80-0454-11eb-9657-9f5408e98396.png)

The mod was only installed client side and has since actually been removed along with the vanilladeathchest.dat files in world>data, DIM1>data and DIM-1>data. I also double checked it's not a waypoint label too. I'm on 1.16.1 and the mod version number is 1.11.0.5-fabric. I'll be quite happy to fix it on my end if you can lend me some info on how the mod places and stores the labels in the game files? Thanks for any help you can offer. It's really bugging me and my players.

**Comment by TheRandomLabs** [comment]
It is not actually VanillaDeathChest that places the labels. VanillaDeathChest simply sets the chest's custom name; another mod must be placing the labels. Can I see a mod list?

**Comment by sean-work** [comment]
Thanks for the quick reply and sorry for the delay of mine! Here is the client-side mod list:
![image](https://user-images.githubusercontent.com/68880081/95004058-38472f80-05de-11eb-9dfe-2328c91b8e61.png)

and here is the server-side mod list:
![image](https://user-images.githubusercontent.com/68880081/95004066-48f7a580-05de-11eb-8782-4b2f91a271d4.png)

Thank you!

Tiny Tweaks mod has an option to display chest names, as well as control the scale of the label, but it doesn't seem to affect the death chest labels at all. Name Pain also has an option to control, player and mob labels but again it doesn't seem to have any impact on labels of death chests. Voxelmap will create a waypoint at a death marker but I have my waypoint labels disabled in world and again, those labels also get removed when i delete or move a waypoint, whereas these linger. I still have the mod "Gravestones" enabled for singleplayer worlds, but that mod puts the player's head on the gravestone rather than generating labels for them, and vanilladeathchest seems to take priority on worlds where it's installed.

**Comment by TheRandomLabs** [comment]
No worries about the delay; I often forget about issues for weeks. 😛 
If you rename a chest in an anvil and place it, does it display a label above it?

**Comment by sean-work** [comment]
Hi yes it does, but the way it displays is entirely different from above. A labelled chest from an anvil displays a text shadow unlike the death chests which display a translucent rectangular backdrop. Also the anvil labelled chest only displays its name while looking at that specific block while also viewing it from the front 180 degree angle. Neither of these two things are true of death chests.
![image](https://user-images.githubusercontent.com/68880081/95200645-a6564700-07d6-11eb-9b58-7a580e4befd3.png)

**Comment by sean-work** [comment]
Okay so I've finally figured it out. The labels in the last picture are from a mod called Tiny Tweaks, toggling the option in the mod setting toggles those. The label display system used by the death chests is from a mod called Chest Tracker, which is primarily used for figuring out where a specific item type is in game via highlighting the chest in the real world, it also does label display, something which I wasn't aware of as i don't label chests myself so was only ever seeing them display on death chests. Since figuring out it's from chest tracker, i have found a way to reduce the label display distance as well as verify that chests still exist in the world and if not, their tracking information (and labels are removed). It's really quick and easy to do this and now that I've set the config for vanilladeathchests to not name death chests, it shouldn't be a future issue. Thanks for your advice! As i doubt I would have figured this out if you hadn't have got me to label a chest myself.
